src/style.css

```
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  width: 100%;
+  background-color: #f00;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.container > :nth-child(2) {
+  background-color: #c00;
+}
+
+.container > :nth-child(3) {
+  background-color: #900;
+}
+
+.container > :nth-child(4) {
+  background-color: #600;
+}
+
+.container > :nth-child(5) {
+  background-color: #300;
+}
+
+.container > :nth-child(6) {
+  background-color: #000;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .block {
+    width: calc(100% / 2);
```

```suggestion
    flex-basis: 50%;
```
Ми можемо користуватись усіма перевагами флекс-контейнерів